Core/src/org/sleuthkit/autopsy/centralrepository/datamodel/PersonaAccount.java

```diff
@@ -379,7 +379,7 @@ static void modifyPersonaAccount(long id, Persona.Confidence confidence, String
             throw new CentralRepoException("Failed to modify persona account, Central Repo is not enabled");
         }
         
-        String updateClause = "UPDATE persona_accounts SET confidence_id = " + confidence.getLevelId() + ", justification = \"" + justification + "\" WHERE id = " + id;
+        String updateClause = "UPDATE persona_accounts SET confidence_id = " + confidence.getLevelId() + ", justification = '" + justification + "' WHERE id = " + id;
```

Comment:
The justification itself could have a single quote in it.
I think we may need to check and escape those.

Reply:
Do we generally do that?  Do we look for other characters that should be escaped from a string?  That doesn't seem like a just here type of issue.  Like what prevents the justification from being an SQL statement?

Reply:
This is something an end user will type in UI. There's a good possibility that it could have a single quote :)

I'm not saying to check it in every string, or check of all kinds of characters.  But a single quote in the justification string will cause the same kind of SQL error you're fixing.  If you search, you will find calls to escapeSingleQuotes() method in Persona API and elsewhere.

Reply:
I get that, my suggestion is that escapeSingleQuotes is not good enough.  What do we do in other places where the user can supply a string field like this?

Reply:
Since it sounds like there is more work in this area might be coming.  I will just remove the other instances of double quotes for this Jira\PR